src/scores/processing/cdf/cdf_functions.py

```diff
@@ -151,13 +151,11 @@ def observed_cdf(
 def integrate_square_piecewise_linear(function_values: xr.DataArray, threshold_dim: str) -> xr.DataArray:
     """Calculates integral values and collapses `threshold_dim`.
 
-    Calculates integral(F(t)^2), where
+    Calculates :math:`\\text{ integral(F(t)^2)}`, where:
```

Comment:
The integral isn't rendering properly. I think you want `\int` instead of `text{ integral}`

Reply:
Fixed, but please check the result. I also fixed the grammar in the first bullet point.